package.json

```diff
@@ -43,7 +43,7 @@
     "aws-sdk": "^2.1.34",
     "broccoli-sass": "^0.6.6",
     "core-object": "^1.1.0",
-    "ember-cli-aptible-shared": "0.0.89",
+    "ember-cli-aptible-shared": "https://github.com/gib/ember-cli-aptible-shared/tarball/role-types",
```

Comment:
This was a temporary for pushing to @fancyremarker's sandbox. Will need to get https://github.com/aptible/ember-cli-aptible-shared/pull/139 merged and update here.